### PR TITLE
Fix error: Cannot execute SQL 'SELECT proacl::text FROM pg_catalog.pg…

### PR DESCRIPTION
**Pull request by glushkovds**
Fix error: Cannot execute SQL 'SELECT proacl::text FROM pg_catalog.pg_proc WHERE proname = ANY (%s) ORDER BY proname, proargtypes': syntax error at or near "%"

### CODE REVIEW

**Comment by Andersson007** [comment]
@glushkovds hello, thanks for the fix! Could you please add a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment) so that users will know about the change?
Thanks

**Comment by glushkovds** [comment]
@Andersson007 hi, done, changelog fragment added.

File: changelogs/fragments/783-fix-postgresql-privs-not-specified-schema.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - "postgresql_privs - #783 fix error: Cannot execute SQL 'SELECT proacl::text FROM pg_catalog.pg_proc WHERE proname = ANY (%s) ORDER BY proname, proargtypes': syntax error at or near %, when trying to grant a function execution and set the schema to not-specified"
```
[the remark this answers is not in the document]
Comment (Andersson007) [reply changelogs/fragments/783-fix-postgresql-privs-not-specified-schema.yml:2]: one bit to make it consistent with other entries

[the remark this answers is not in the document]
Comment (Andersson007) [reply changelogs/fragments/783-fix-postgresql-privs-not-specified-schema.yml:2]: if you agree, after clicking "Commit suggesting" I'd be happy to merge the PR

**Comment by Andersson007** [comment]
@glushkovds thanks for the contribution!

**Pull request merged**